### PR TITLE
fix(DTFS2-7577): fix console logging in gef api error handler

### PR DESCRIPTION
**Pull request by AlexBramhill**
## Introduction :pencil2:
adds better logs around our api error handling

### CODE REVIEW

**Review by ttbarnes — DISMISSED** [review]
Great work

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Thanks for this @AlexBramhill this PR can be now closed.

**Comment by AlexBramhill** [comment]
> Thanks for this @AlexBramhill this PR can be now closed.

Are you sure we don't still need this? We should probably log all non OK errors?

File: gef-ui/server/utils/helpers.js [hunk 1]
```diff
@@ -35,6 +35,8 @@ const isMIAWithoutChangedToIssuedFacilities = (app) =>
 
 // Converts Api errors into more manageable objects
 const apiErrorHandler = ({ code, response }) => {
+  console.error({ code, response });
```
Comment (abhi-markan) [inline gef-ui/server/utils/helpers.js:38]:
```suggestion
  console.error('API error %s %o', code, response);
```

**Review by abhi-markan — CHANGES_REQUESTED** [review]

**Review by ttbarnes — APPROVED** [review]

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**